extras/config/default-config.json

```diff
@@ -50,5 +50,12 @@
   "nmea2000": {
     "txEnabled": true,
     "rxEnabled": true
+  },
+  "performance": {
+    "enabled": true,
+    "boatSpeedCorrectionTable": "boatspeedCorr.cal"
```

Comment:
missing a comma at the end of the line. this will not load properly.

Reply:
done